dev/com.ibm.ws.org.apache.commons.collections4/bnd.bnd

```diff
@@ -0,0 +1,15 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= jar:${fileuri;${repo;org.apache.commons:commons-collections4;4.4;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
+
+-buildpath: org.apache.commons:commons-collections4;version=4.4
```

Comment:
This is a 2.5MB jar, we probably can't justify adding it as a dependency. Also, when adding a bundle you also need to reference it in the feature files.

It should be easy enough to duplicate the functionality of `PeekingIterator`.

Reply:
We actually ship apache collections (not collections4) already. I was planning to see about moving us to the later version in a follow-up.